src/Translation/TranslatorHelper.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mathematicator\Search\Translation;
+
+
+use Symfony\Component\Translation\Loader\YamlFileLoader;
+use Symfony\Component\Translation\Translator;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class TranslatorHelper
+{
+
+	/**
+	 * @var TranslatorInterface
+	 */
+	private $translator;
+
+
+	/**
+	 * @var bool
+	 */
+	private $isInitialized = false;
+
+
+	/**
+	 * Available translations ordered by priority
+	 *
+	 * @var string[]
+	 */
+	private $languages = ['cs_CZ', 'en_US'];
+
+
+	/**
+	 * If no translation is available, than this languages are used instead.
+	 *
+	 * @var string[]
+	 */
+	private $fallbackLanguages = ['en_US', 'cs_CZ'];
+
+
+	/**
+	 * TranslatorHelper constructor.
+	 * @param Translator $translator
+	 */
+	public function __construct(
+		Translator $translator
+	)
```

Comment:
Codestyle should be `public function __construct(Translator $translator)`

Reply:
That's really pain since there is no Codestyle spec for IDE.

Reply:
I don't think I can accept code that breaks tests no matter how it originated.

Reply:
I don't talk about you should accept it. Anytime I will send a PR and some tests fails, I'll try to fix it, sure. I just say that it's pain.